app/controllers/experiments_controller.rb

```diff
@@ -535,6 +535,10 @@ def experiment_moes
     render json: moes_info
   end
 
+  def experiment_results
```

Comment:
Zdecydowanie wolę tego tak nie zostawiać, bo po takiej nazwie spodziewałbym się, że odpowiedź to sam JSON spod pola results.